Formula/cloudflare-wrangler.rb

```diff
@@ -0,0 +1,18 @@
+class CloudflareWrangler < Formula
+  desc "Wrangler is a CLI tool designed for Cloudflare Workers"
+  homepage "https://developers.cloudflare.com/workers/tooling/wrangler/"
+  url "https://github.com/cloudflare/wrangler/archive/v1.8.0.tar.gz"
+  sha256 "ebb31c2549733991823f965025da493bbc2a1644d02668549700793a29df0229"
+  head "https://github.com/cloudflare/wrangler.git"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "."
+  end
+
+  test do
+    system "#{bin}/wrangler", "--help"
+    assert_match "wrangler " + version.to_s, shell_output("#{bin}/wrangler -V").chomp
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
Same if the software requires a virtual machine, docker instance, etc. to be running.